com.unity.ml-agents/Runtime/Policy/BehaviorParameters.cs

```diff
@@ -37,7 +36,7 @@ private enum BehaviorType
         string m_BehaviorName = "My Behavior";
         [HideInInspector]
         [SerializeField]
-        public int m_TeamID = 0;
+        public int m_TeamID;
```

Comment:
This and m_useChildSensors are still public with `m_` prefixes. Not sure if we should rename or make internal. `m_TeamID` seems like it needs to be mutable at runtime.